msu/hooks/items/item.nut

```diff
@@ -60,10 +60,11 @@
 			}
 		}
 
-		return names.len() > 1 ? "[color=" + ::Const.UI.Color.NegativeValue + "]" + names.slice(0, -2) + "[/color]\n\n" + getDescription() : getDescription();
+		if (names != "") names = names.slice(0, -2);
```

Comment:
This change is wrong. The original code is fine as far as I can see unless I am missing something?

Reply:
the one thing I think we should change would be instead of `names.len() > 1` we should use `names != ""` since I you measured the performance to be 2x better (I just double checked this and yeah that is correct), but other than that yeah looking back I don't see what the problem is with this portion

Reply:
Yeah this didn't need that check, my bad